src/caffe/layers/batch_norm_layer.cpp

```diff
@@ -22,17 +23,29 @@ void BatchNormLayer<Dtype>::LayerSetUp(const vector<Blob<Dtype>*>& bottom,
   if (this->blobs_.size() > 0) {
     LOG(INFO) << "Skipping parameter initialization";
   } else {
-    this->blobs_.resize(3);
+    this->blobs_.resize(5);
```

Comment:
blobs_[4] is referenced in a few places, but I agree that it looks like it's not used anywhere.

@borisgin can you comment on this?